GrabMigrator/GrabMigrator.cs

```diff
@@ -8,60 +8,61 @@
 using System.Text.Json;
 using System.Text.RegularExpressions;
 using Nuke.Common;
+using VirtoCommerce.Platform.Core.Common;
 
 namespace GrabMigrator
 {
     /// <summary>
-    /// Grab-migrator target implementation
+    ///     Grab-migrator target implementation
     /// </summary>
     internal class GrabMigrator
     {
         public void Do(string configFilePath)
```

Comment:
`Do` method has huge cognitive complexity, so reading this code is a real pain. Suggest to use guard clause for `if (!string.IsNullOrEmpty(configFilePath))`  and `if (File.Exists(configFilePath))`, etc. Extract methods from `config?,Grab == true` and `if (config?.Apply == true)`;